### PR TITLE
insights-agent: unpin patch versions of a few images

### PR DESCRIPTION
**Pull request by rbren**

### CODE REVIEW

File: stable/insights-agent/README.md [hunk 1]
```diff
@@ -60,9 +60,9 @@ Parameter | Description | Default
...
 `workloads.timeout` | Maximum time in seconds to wait for the report | 60
 `workloads.image.repository` | Repository to use for the workload image | quay.io/fairwinds/workloads
-`workloads.image.tag` | Image tag to use for the Workloads image | 1.0.1
+`workloads.image.tag` | Image tag to use for the Workloads image | 1.0
 `trivy.enabled` | Enable Trivy container scanning reports | true
 `trivy.schedule` | Cron expression for running the trivy report | `rand * * * *`
 `trivy.timeout` | Maximum time in seconds to wait for the report | 3600
 `trivy.image.repository` | Repository to use for the workload image | quay.io/fairwinds/fw-trivy
-`trivy.image.tag` | Image tag to use for the Workloads image | 0.0.1
+`trivy.image.tag` | Image tag to use for the Workloads image | 0.0
```
Comment (makoscafee) [inline stable/insights-agent/README.md]:
```suggestion
`trivy.image.tag` | Image tag to use for the Trivy image | 0.0
```

File: stable/insights-agent/README.md [hunk 2]
```diff
@@ -60,9 +60,9 @@ Parameter | Description | Default
 `workloads.schedule` | Cron expression for running the workloads report | `rand * * * *`
 `workloads.timeout` | Maximum time in seconds to wait for the report | 60
 `workloads.image.repository` | Repository to use for the workload image | quay.io/fairwinds/workloads
-`workloads.image.tag` | Image tag to use for the Workloads image | 1.0.1
+`workloads.image.tag` | Image tag to use for the Workloads image | 1.0
 `trivy.enabled` | Enable Trivy container scanning reports | true
 `trivy.schedule` | Cron expression for running the trivy report | `rand * * * *`
 `trivy.timeout` | Maximum time in seconds to wait for the report | 3600
 `trivy.image.repository` | Repository to use for the workload image | quay.io/fairwinds/fw-trivy
```
Comment (makoscafee) [inline stable/insights-agent/README.md]:
```suggestion
`trivy.image.repository` | Repository to use for the Trivy image | quay.io/fairwinds/fw-trivy
```

File: stable/insights-agent/README.md [hunk 3]
```diff
@@ -60,9 +60,9 @@ Parameter | Description | Default
 `workloads.schedule` | Cron expression for running the workloads report | `rand * * * *`
 `workloads.timeout` | Maximum time in seconds to wait for the report | 60
 `workloads.image.repository` | Repository to use for the workload image | quay.io/fairwinds/workloads
-`workloads.image.tag` | Image tag to use for the Workloads image | 1.0.1
+`workloads.image.tag` | Image tag to use for the Workloads image | 1.0
```
Comment (makoscafee) [inline stable/insights-agent/README.md]:
```suggestion
`workloads.image.tag` | Image tag to use for the workloads image | 1.0
```

consistency

File: stable/insights-agent/README.md [hunk 4]
```diff
@@ -60,9 +60,9 @@ Parameter | Description | Default
...
 `trivy.enabled` | Enable Trivy container scanning reports | true
 `trivy.schedule` | Cron expression for running the trivy report | `rand * * * *`
```
Comment (makoscafee) [inline stable/insights-agent/README.md]:
```suggestion
`trivy.schedule` | Cron expression for running the Trivy report | `rand * * * *`
```
Consistency

**Review by makoscafee — CHANGES_REQUESTED** [review]
LGTM. just a few nits

**Comment by rbren** [comment]
Good catches!

**Review by makoscafee — APPROVED** [review]

**Pull request merged**